llvm/docs/ProgrammersManual.rst

```diff
@@ -1384,9 +1382,28 @@ So if executed on the following code:
 
 It would delete number ``%2`` and ``%3``.
 
-A utility is provided in `utils/bisect-skip-count` to binary search
-skip and count arguments. It can be used to automatically minimize the
-skip and count for a debug-counter variable.
+A utility is provided in `llvm/tools/delta-driver/delta-driver.cpp` to drive the automated delta debugging.
```

Comment:
So I agree that it looks like your delta-driver supersedes the functionality of bisect-skip-count; I don't think we just want to delete the latter outright, since people may be used to using it, but please add a comment there that encourages people to use the more general / more powerful tool.

Reply:
Tell me if the update is ok for you.